osu.Game/Storyboards/StoryboardVideo.cs

```diff
@@ -3,23 +3,18 @@
 
 using osu.Framework.Graphics;
 using osu.Game.Storyboards.Drawables;
+using osuTK;
 
 namespace osu.Game.Storyboards
 {
-    public class StoryboardVideo : IStoryboardElement
+    public class StoryboardVideo : StoryboardSprite
     {
-        public string Path { get; }
-
-        public bool IsDrawable => true;
-
-        public double StartTime { get; }
-
         public StoryboardVideo(string path, double offset)
+            : base(path, Anchor.Centre, Vector2.Zero)
         {
-            Path = path;
-            StartTime = offset;
+            TimelineGroup.Alpha.Add(Easing.None, offset, offset, 0, 1);
```

Comment:
I'm not sure what this does given that the transforms from this aren't actually applied on the drawable, as per above. But aside from that, I do wonder how this is gonna work if the video has actual user-specified alpha transforms on it.

Reply:
This can be removed I believe. The equivalent is already done in `LoadComplete`. Left it behind after finding the relevant fades there.

Reply:
On revisiting, I *think* this is required to set the correct `StartTime` for the final usage to generate the transforms

https://github.com/ppy/osu/blob/7b92c725b1e4004d1e082989522906335efe4859/osu.Game/Storyboards/StoryboardSprite.cs#L26

https://github.com/ppy/osu/blob/17ca29c2c6e0e5f141f5393923a511cfa2d61437/osu.Game/Storyboards/Drawables/DrawableStoryboardVideo.cs#L63-L71

Reply:
Well I see at least two problems here, one major one minor...

- The major one is that on all storyboards with videos the videos appear to fade in instantly now. That second fade you linked there appears to be just dead. Feels pretty bad.
- The minor one is that fade commands don't work on video sprites at all. This can probably be dismissed until someone inevitably drags in an example of a storyboard that uses this.

Reply:
I've fixed the fade in not working. The other issue can be ignored for now, I agree.